lib/pronto/golang/tools/base.rb

```diff
@@ -30,7 +30,7 @@ def installed?
       end
 
       def enabled?
-        @config.fetch('enabled', true) # Default to true if the key is not configured
+        !!@config['enabled']
```

Comment:
`fetch` was used here on purpose, to ensure that the key is set and to make the default value more obvious, if it is not.
This also changes the previous behaviour from being enabled by default to being disabled by default.
That's something that should be discussed independently of this change